src/infobox.tsx

```diff
@@ -211,17 +211,10 @@ const Shares = (props: ShareProps): JSX.Element => {
 
   useEffect(() => {
     const getGrantees = async (): Promise<any> => {
-      const home: any = await requestAPI<any>('/api/cs3/file/home', {
-        method: 'GET'
-      });
-
-      const resource =
-        home.path +
-        '/' +
-        props.fileInfo.path
-          .replace('cs3driveShareByMe:', '')
-          .replace('cs3drive:', '')
-          .replace('cs3driveShareWithMe:', '');
+      const resource = props.fileInfo.path
```

Comment:
Why isn't this info removed from the backend instead? If I understand correctly, you already assume the drives are not exposed to the frontend, no?

Reply:
After all there are no changes in this file. This is a matter of formatting, the replacing above is in master for months.

Reply:
After all there are no changes in this file. This is a matter of formatting, the replacing above is in master for months.

Reply:
I understand it was there, it was necessary, I just wonder with the changes done in the backend if this was address or not. But it's fine to keep it for a bit longer.

Reply:
@mateusz-garlacz please let me know it it's necessary, otherwise let's merge.

Reply:
@diocas it is still necessary to keep it